app/src/main/java/dev/arkbuilders/navigator/presentation/screen/resources/adapter/ResourcesGridPresenter.kt

```diff
@@ -145,8 +145,16 @@ class ResourcesGridPresenter(
                 return@launch
         }
 
+//        router.navigateToFragmentUsingAdd(
```

Comment:
Remove commented out code

Reply:
This package will be removed after I merge this PR. No worries

Reply:
@hieuwu I think it's better to remove comments before merging. If we want to restore older version we can just checkout older commit..